lib/gringotts/gateways/trexle.ex

```diff
@@ -39,7 +39,7 @@ defmodule Gringotts.Gateways.Trexle do
   use Gringotts.Gateways.Base
   use Gringotts.Adapter, required_config: [:api_key, :default_currency]
```

Comment:
@oyeb since we are using money struct, is `:default_currency` is still required.

Reply:
No, it's not required.Will remove it from required config.